tests/BasicSearchTests.FunctionalTests.Core/StartupFunctionalTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+using System;
+using System.Net.Http;
+using BasicSearchTests.FunctionalTests.Core.Models;
+using Newtonsoft.Json;
+
+namespace BasicSearchTests.FunctionalTests.Core
+{
+    public class StartupFunctionalTests
+    {
+        private HttpClient _client;
+        private static string RegistrationBaseUrl = "RegistrationsBaseUrl";
+
+        public StartupFunctionalTests()
+        {
+            // Arrange
+            _client = new HttpClient { BaseAddress = new Uri(EnvironmentSettings.SearchServiceBaseUrl) };
```

Comment:
I think its an overkill for simple things. If we had more common functionality I'd agree. #wontfix